datalad/support/sshconnector.py

```diff
@@ -158,6 +162,40 @@ def open(self):
     def close(self):
         raise NotImplementedError
 
+    @property
+    def ssh_command(self):
+        """determine which ssh client should be used.
+
+        On Windows OpenSSH for Windows is required. It is searched in its
+        default location. This can be overridden by setting the datalad
+        config variable 'datalad.windows.ssh.client' to the path of the
+        system ssh client.
+        """
+        if self._ssh_command:
+            return self._ssh_command
```

Comment:
see https://github.com/datalad/datalad/issues/6556 -- might be time to start using `@cached_property` but we better centralize its import (needs try/except for python < 3.8) e.g. in `datalad.utils`?

Reply:
I would do that in a different PR!?